client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/survey/index.tsx

```diff
@@ -0,0 +1,42 @@
+import { useLocale } from '@automattic/i18n-utils';
+import Banner from 'calypso/components/banner';
+import './survey.scss';
+
+interface Props {
+	setSurveyDismissed: ( dismissed: boolean ) => void;
+}
+
+const Survey = ( { setSurveyDismissed }: Props ) => {
+	const locale = useLocale();
+
+	if ( ! locale.includes( 'en' ) ) {
+		return null;
+	}
+
+	return (
+		<Banner
+			className="pattern-assembler__survey"
+			title={
+				<>
+					<a href="https://lucasmdo.survey.fm/assembler-survey-modal" target="blank">
+						Fill out this quick survey
+					</a>{ ' ' }
+					and help us to improve our product.
+				</>
+			}
+			event="assembler-june-2023"
+			onDismiss={ ( e: Event ) => {
+				e.stopPropagation();
+				setSurveyDismissed( true );
+			} }
+			dismissTemporary
```

Comment:
It's odd that I'm seeing the banner-as-link, I've tried different browsers and disabled cache, and it's the same 🤔
Could we add `disableHref` just in case?
```suggestion
			dismissTemporary
			disableHref
```

Reply:
I think `disableHref` is not related to the issue you are facing because the right arrow seems to be added because [showLinkIcon](https://github.com/Automattic/wp-calypso/blob/df14f3f4e8333002d640409e532a1623fe854036/packages/components/src/card/index.tsx#L57) is `true` [by default](https://github.com/Automattic/wp-calypso/blob/df14f3f4e8333002d640409e532a1623fe854036/packages/components/src/card/index.tsx#L38). 

However, `Banner` doesn't pass it to `<Card>` so I updated that to make sure it's `false`.

Can you please test again when the the new Calypos Live image is ready?

Reply:
I still see the banner as a link. 👀  It redirects me to `/plans/<site>` by clicking the banner.

https://github.com/Automattic/wp-calypso/assets/5287479/232e8c1d-547d-4a30-8098-ca9e32d638a6

Reply:
Same to what @okmttdhr said, the arrow icon is gone but the banner is still clickable and landing in the plans page.

Reply:
Are you both creating a new site to test? Not sure what else can be different because when I test using a new site it works fine.


https://github.com/Automattic/wp-calypso/assets/1881481/071320bd-f7bf-45bb-b813-709446f28213

Reply:
I'm made a change to pass `disableHref: true` because the issue is in the [condition](https://github.com/Automattic/wp-calypso/blob/df14f3f4e8333002d640409e532a1623fe854036/client/components/banner/index.jsx#L102) that checks that your site can upgrade and has the slug [here](https://github.com/Automattic/wp-calypso/blob/df14f3f4e8333002d640409e532a1623fe854036/client/components/banner/index.jsx#L341)

Reply:
Nice!